website/app/validators.php

```diff
@@ -24,7 +24,10 @@
 }, _('This language is not supported'));
 
 $validator->addValidator('ciphered_password', function ($field, $input, $param = null) {
-    return substr($input[$field], 0, 7) === '$2a$11$';
+    $expectedPasswordPrefix = '$2a$'.sprintf("%02d", GK_PASSWORD_HASH_ROTATION).'$';
```

Comment:
`sprintf('$2a$%02d$', GK_PASSWORD_HASH_ROTATION)` should work, and is so nicer :) (And no double quote, else Codacy will complain - and it's not optimized)

Reply:
good catch, was my brain fault